addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1532,6 +1549,12 @@ const Wysiwyg = Widget.extend({
         if (!options.snippets) {
             $toolbar.find('#justify, #media-insert').remove();
         }
+        $toolbar.find('#image-fullscreen').click(() => {
+            if (!this.lastMediaClicked || !this.lastMediaClicked.src) {
```

Comment:
Why no `?.` when you have used it multiple times before in the same commit ? :p

Reply:
I guess the negative threw me off ^^'